agent/pkg/chassis/controller/controller.go

```diff
@@ -82,7 +82,7 @@ func (c *ChassisController) epUpdate(oldObj, newObj interface{}) {
 	// service, endpoints and destinationRule should have the same name
 	dr, err := c.drLister.DestinationRules(ep.Namespace).Get(ep.Name)
 	if err == nil && dr != nil && isConsistentHashLB(dr) {
-		key := fmt.Sprintf("%s.%s", ep.Namespace, ep.Name)
+		key := fmt.Sprintf("%s.%s", ep.Name, ep.Namespace)
```

Comment:
if the service key format doesn't causes bug, I don't agree to fix the service key format, unless you did comprehensive functional test